pkg/cas/content_addressable_storage_server_test.go

```diff
@@ -0,0 +1,88 @@
+package cas_test
+
+import (
+	"context"
+	"testing"
+
+	remoteexecution "github.com/bazelbuild/remote-apis/build/bazel/remote/execution/v2"
+	"github.com/buildbarn/bb-storage/internal/mock"
+	"github.com/buildbarn/bb-storage/pkg/blobstore/buffer"
+	"github.com/buildbarn/bb-storage/pkg/cas"
+	"github.com/buildbarn/bb-storage/pkg/digest"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/require"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+func TestContentAddressableStorageServerBatchReadBlobsSuccess(t *testing.T) {
+	ctrl, ctx := gomock.WithContext(context.Background(), t)
+	defer ctrl.Finish()
+
+	digest1 := digest.MustNewDigest("ubuntu1804", "0000000000000000000000000000000000000000000000000000000000000001", 123)
```

Comment:
With the unit tests I always try to make them representative of what may happen in real life. Because many of the tests use `NewValidatedBufferFromByteSlice()`, we can get away with using digests that don't match up with the data, but it's not what may be possible in practice.

Could you please change these digests to the following?
```go
digest1 := digest.MustNewDigest("ubuntu1804", "409a7f83ac6b31dc8c77e3ec18038f209bd2f545e0f4177c2e2381aa4e067b49", 123)
digest2 := digest.MustNewDigest("ubuntu1804", "0479688f99e8cbc70291ce272876ff8e0db71a0889daf2752884b0996056b4a0", 234)
```